flow/scenarios/loop.py

```diff
@@ -196,7 +195,20 @@ def specify_edge_starts(self):
         """See parent class."""
         edgelen = self.net_params.additional_params["length"] / 4
 
-        edgestarts = [("bottom", 0), ("right", edgelen), ("top", 2 * edgelen),
-                      ("left", 3 * edgelen)]
+        edgestarts = [("bottom", 0),
+                      ("right", edgelen + 0.1),
+                      ("top", 2 * edgelen + 0.2),
+                      ("left", 3 * edgelen + 0.3)]
+
+        return edgestarts
+
+    def specify_internal_edge_starts(self):
+        """See parent class."""
+        edgelen = self.net_params.additional_params["length"] / 4
```

Comment:
Same as above. Not really fair to through the division by 4 here and above on you, but since you're making the changes anyways, it'd be good to deal with that magic number as well.